dndwiki/classes/class-gunslinger.xml

```diff
@@ -0,0 +1,874 @@
+    </description>
+    <sheet>
+      <description>You gain proficiency in the Religion skills.</description>
+    </sheet>
+    <rules>
+      <grant type="Proficiency" id="ID_PROFICIENCY_SKILL_RELIGION" />
+      <stat name="religion:proficiency" value="Proficiency" bonus="double" />
+    </rules>
+  </element>
+  <element name="Ranged Smite" type="Archetype Feature" source="D&amp;D Wiki" id="ID_WIKI_ARCHETYPE_FEATURE_RANGED_SMITE">
+    <description>
+      <p>Beginning at 3rd level, your unique blend of martial and divine training allows you to "project" your smiting spells at a distance. A spell with "Smite" in the name that can normally only be used with a melee weapon can now be used with a firearm, but only if the target is within normal range. When using a firearm this way, being within 5 feet of a hostile creature doesn't impose disadvantage on your ranged attack rolls.</p>
+    </description>
+    <sheet>
+      <description>Spell with the word "smite" can be used with your ranged weapons and dont get disadvantage attacking a creature in melee range.</description>
+    </sheet>
+  </element>
+  <element name="Bane Shot" type="Archetype Feature" source="D&amp;D Wiki" id="ID_WIKI_ARCHETYPE_FEATURE_BANE_SHOT">
+    <description>
+      <p>Starting at 7th level, attacks with your firearms deal 2d6 additional radiant damage against fiends and undead</p>
+    </description>
+    <sheet>
+      <description>Firearm attacks deal 2d6 additional radiant damage against fiends and undead.</description>
+    </sheet>
+    <rules />
+  </element>
+  <element name="Holy Aura" type="Archetype Feature" source="D&amp;D Wiki" id="ID_WIKI_ARCHETYPE_FEATURE_HOLY_AURA">
+    <description>
+      <p>Once you reach 15th level, you become permanently protected by a protection from evil and good spell which cannot be suppressed or dispelled.</p>
+    </description>
+    <sheet>
+      <description>You become permanently protected by a protection from evil and good spell which cannot be suppressed or dispelled.</description>
+    </sheet>
+  </element>
+  <element name="Shot of Disruption" type="Archetype Feature" source="D&amp;D Wiki" id="ID_WIKI_ARCHETYPE_FEATURE_SHOT_OF_DISRUPTION">
+    <description>
+      <p>Beginning at 18th level, your firearm attacks are devastatingly powerful against fiends and undead, sundering the essence of wicked beings with every shot. If a fiend or undead has 50 hit points or fewer after taking damage from one of your firearm attacks, it must succeed on a Wisdom saving throw (DC = 8 + your Charisma modiier + your proficiency bonus) or be destroyed. On a successful save, the creature becomes frightened of you until the end of your next turn.</p>
+    </description>
+    <sheet>
+      <description>Fiends or undead under 50 HP that are hit by your firearm attacks have to make a wisdom save (DC {{trick_shot:dc}}) or be killed immediately, on a sucess the creatured if frightened until the end of your next turn.</description>
```

Comment:
1. again, `{{trick_shot:dc}}` uses WIS mod not CHA mod, so it should be changed to new stat.
2. two typos in `on a sucess the creatured`